tests/helpers.ts

```diff
@@ -393,8 +392,11 @@ export function deepTemplateReplace(
 			if (Array.isArray(data)) {
 				return data.map((i) => deepTemplateReplace(i, templateVars));
 			}
-			return _.mapValues(data, (value) =>
-				deepTemplateReplace(value, templateVars),
+			return Object.fromEntries(
+				Object.entries(data).map(([key, value]) => [
+					key,
+					deepTemplateReplace(value, templateVars),
+				]),
 			);
```

Comment:
I think this might be better as
```suggestion
			for (const key of Object.keys(data)) {
				data[key] = deepTemplateReplace(data[key], templateVars);
			}
			return data;
```
because it will avoid so many intermediate allocations, assuming we don't mind mutating the input object (I suspect we don't mind, but it is the main concern)